packages/sanity/src/core/components/Hotkeys.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {type HTMLProps, type RefAttributes} from 'react'
+import {type HotkeysProps as UIHotkeysProps, Hotkeys as UIHotkeys} from '@sanity/ui'
+
+/**
+ * Properties for the `Hotkeys` component.
+ *
+ * @public
+ */
+export type HotkeysProps = UIHotkeysProps & {
+  /**
+   * Whether to make the keys platform-aware (eg `alt` to `option` on Apple devices).
+   *
+   * @defaultValue true
+   */
+  makePlatformAware?: boolean
```

Comment:
out of curiosity and lack of imagination: in what cases would you want to make it not platform aware?

Reply:
Potentially if showing both alternatives, eg something like:
```
Windows: Ctrl+Alt+I
Mac: Ctrl+Opt+I
```